#!/usr/bin/env python
PACKAGE = "synchronizer_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Servo_Position",    int_t,    0, "Servo Rotation Position", 64,  30, 64)

exit(gen.generate(PACKAGE, "synchronizer_ros", "Servo"))